app/code/core/Mage/Core/Model/Resource/Abstract.php

```diff
@@ -44,6 +56,24 @@ public function __construct()
         $this->_construct();
     }
 
+    /**
+     * Default transaction isolation level defined in the database.
+     *
+     * @todo Default this value with whatever the database reports in SELECT @@GLOBAL.TX_ISOLATION
+     *
+     * @var int
+     */
+    static protected $_defaultIsolationLevel = self::READ_COMMITTED;
```

Comment:
Maybe make this configurable with local.xml instead? Then it is configurable via the application and not from the database.